services/task-svc/api/patient-svc.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+option go_package = "task-svc/api";
+
+//
+// CreatePatient
+//
+
+message CreatePatientRequest {
+	string humanReadableIdentifier = 1;
```

Comment:
don't we need any kind of validation here? I think this will accept an empty string or, by extend a missing value. please check for all other requests as well. [see this example](https://github.com/helpwave/services/blob/main/services/auth-svc/api/auth-svc.proto#L21)